packages/builders/plugins/jest/resolver.ts

```diff
@@ -37,6 +37,16 @@ function getCompilerSetup(rootDir: string) {
 let compilerSetup;
 
 module.exports = function(path: string, options: ResolveOptions) {
+  const ext = extname(path);
+  if (
+    ext === '.css' ||
+    ext === '.scss' ||
+    ext === '.sass' ||
+    ext === '.less' ||
+    ext === '.styl'
+  ) {
+    return require.resolve('identity-obj-proxy');
```

Comment:
why is this necessary?

Reply:
react apps expect styles to be an object when imported? Not totally sure.. @jaysoo has more info.